### PR TITLE
No more upload_allowed

### PR DESCRIPTION
**Pull request by jobevers**
This is unused and largely polluting the argument list of too many functions.

### CODE REVIEW

**Comment by jobevers** [comment]
will merge this one later, after some of the other changes are better tested

**Comment by jobevers** [comment]
@jackrobison please review

File: lbrynet/lbrylive/client/LiveStreamDownloader.py [hunk 1]
```diff
@@ -161,7 +161,6 @@ def create_downloader(stream_hash):
             stream_downloader = FullLiveStreamDownloader(stream_hash, self.peer_finder, self.rate_limiter,
                                                          self.blob_manager, self.stream_info_manager,
                                                          prm, self.wallet, True)
```
Comment (jackrobison) [inline lbrynet/lbrylive/client/LiveStreamDownloader.py]: This one still has an upload_allowed in it

**Comment by jackrobison** [comment]
Can you rebase on master?

File: lbrynet/lbryfilemanager/EncryptedFileManager.py [hunk 2]
```diff
@@ -266,9 +266,9 @@ def _open_db(self):
     @rerun_if_locked
     def _save_lbry_file(self, stream_hash, data_payment_rate):
         def do_save(db_transaction):
-            db_transaction.execute("insert into lbry_file_options values (?, ?, ?)",
-                                  (data_payment_rate, ManagedEncryptedFileDownloader.STATUS_STOPPED,
-                                   stream_hash))
+            row = (data_payment_rate, ManagedEncryptedFileDownloader.STATUS_STOPPED, stream_hash)
+            log.debug('Saving %s to db', row)
```
Comment (jobevers) [inline lbrynet/lbryfilemanager/EncryptedFileManager.py]: I can take out this log statement - I put it in temporarily for debugging

File: tests/functional/test_misc.py [hunk 3]
```diff
@@ -742,7 +737,7 @@ def create_downloader(metadata, prm):
             factories = metadata.factories
             chosen_options = [
                 o.default_value for o in options.get_downloader_options(info_validator, prm)]
-            return factories[0].make_downloader(metadata, chosen_options, prm)
+            return factories[0].make_downloader(metadata, choosen_options, prm)
```
Comment (jobevers) [inline tests/functional/test_misc.py]: fix typo

Comment (jobevers) [inline tests/functional/test_misc.py]: actually, I can revert all the changes to this file...

**Pull request merged**